project-reports/2024/2024-Q2-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: 2024 Q42 Hyperledger FireFly
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+Hyperledger FireFly successfully launched [V1.3](https://www.hyperledger.org/blog/hyperledger-firefly-v1.3-is-now-available) this quarter. This is a significant upgrade that makes the industry’s first open-source Web3 Gateway an even more powerful platform for tokenization, multi-chain interoperability, and building blockchain-based applications.
+
+FireFly continues to expand its maintainer base and diversity with 3 new maintainers this quarter. New maintainers include Enrique Lacal from Kaleido, Sam May from Kaleido, and Alexey Semenyuk from Instinct Tools.
+
+In addition, FireFly continues to power many use cases in the digital assets community and remains a leader in enterprise middleware.
+
+### Community adoption updates
+
+- [SWIFT CBDC Phase 2](https://www.swift.com/sites/default/files/files/swift_cbdc_sandbox_project_results_report_phase2_final_220324.pdf)
+  - Hyperledger Firefly was used to facilitate interoperability in the world’s largest CBDC industry collaboration to date.
+  - Interoperability was tested between 38 different banks across chains running Besu, Fabric, and Corda.
+  - Networks included 7 simulated CBDC networks, a foreign exchange network, a digital asset network, and a simulated CLS application, representing a mix of digital and traditional networks.
+
+
+Community members have given talks featuring Hyperledger FireFly at a number of events including:
+
+- Consensus 2024 where Hyperledger FireFly members presented at the Hyperledger booth
+- Hyperledger Member Webinar: [Streamlining Delivery-versus-Payment with Hyperledger Technologies](https://www.youtube.com/watch?v=gnZOXHBsVdk&t=2737s)
+- Duke Web3 Conference: [Real World Asset Tokenization](https://www.dukeweb3conference.com/speakers)
+- Token 2049: [Decentralized Trust & Web3 in the Enterprise](https://lu.ma/lxjqfuw3?hss_channel=lcp-10851358)
+- University of Southampton Business School lecture: [Web3 in Industry](https://www.linkedin.com/feed/update/urn:li:activity:7196499141403856896)
+- [FireFly 1.3 webinar](https://www.meetup.com/hyperledger-raleigh/events/301138155/) scheduled for 6/10
+
+### Contributor activity updates
+
+The star activity for the FireFly repository has continued to grow from 430 end of last quarter to 481 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/HLFF_2024_Q2_stars.png "image_tooltip")
```

Comment:
Looks like path does not match location of file.

Reply:
The `images` folder was pushed to `2023` instead of `2024`.